CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [BUG] Remove duplicate sample data as id 90943e30-9a47-11e8-b64d-95841ca0b247 ([5668](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5668))
 - [BUG][Multiple Datasource] Fix datasource testing connection unexpectedly passed with wrong endpoint [#5663](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5663)
 - [Table Visualization] Fix filter action buttons for split table aggregations ([#5619](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5619))
+- [BUG] [Multiple Datasource] datasource id is required if there is no open-search hosts config ([#5882](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5882))
```

Comment:
Should we start with `Datasource id xxx` to be consistent with rest of the title? Also, should it be the same as the PR title?

Reply:
@xinruiba How about "**Improved error handling for the search API when a null value is passed for the dataSourceId.**"? If it looks good to you, change the PR title as well as changelog.

Reply:
Sounds good, will update it.